inference-engine/tests/unit/gna/ngraph/transformations/gna_split_convolution_with_large_buffer_size.cpp

```diff
@@ -0,0 +1,597 @@
+        auto split_node_c2 = ngraph::opset7::Constant::create(ngraph::element::i64, ngraph::Shape({5}), {960, 960, 960, 960, 256});
+        auto split_node = std::make_shared<ngraph::opset7::VariadicSplit>(input_params,
+                                                                            split_node_c1,
+                                                                            split_node_c2);
+
+        auto weights = ngraph::opset1::Constant::create(ngraph::element::f32,
+                                                        ngraph::Shape{1, 64, 1, 1}, {1});
+        auto bias = ngraph::opset1::Constant::create(ngraph::element::f32,
+                                                     ngraph::Shape{1, 1, 1}, {1});
+
+        ngraph::OutputVector output_fq_operations;
+        for (int i = 0; i < 5; ++i) {
+            auto convolution_operation = std::make_shared<ngraph::opset7::Convolution>(split_node->output(i),
+                                                                                weights,
+                                                                                ngraph::Strides{1, 1},
+                                                                                ngraph::CoordinateDiff{0, 0},
+                                                                                ngraph::CoordinateDiff{0, 0},
+                                                                                ngraph::Strides{1, 1});
+            auto input_low = ngraph::opset7::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {1});
+            auto input_high = ngraph::opset7::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {20});
+            auto output_low = ngraph::opset7::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {0});
+            auto output_high = ngraph::opset7::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {10});
+            auto fake_quantize_op = std::make_shared<ngraph::opset7::FakeQuantize>(convolution_operation, input_low,
+                                                                                input_high, output_low,
+                                                                                output_high, 11);
+            output_fq_operations.push_back(fake_quantize_op);
+        }
+
+        auto concat = std::make_shared<ngraph::opset7::Concat>(output_fq_operations, 3);
+
+        auto result = std::make_shared<ngraph::opset7::Result>(concat);
+        reference_func = std::make_shared<ngraph::Function>(ngraph::ResultVector{result},
+                                              ngraph::ParameterVector{input_params});
+    }
+    const FunctionsComparator func_comparator = FunctionsComparator::with_default().enable(FunctionsComparator::ATTRIBUTES);
+    const FunctionsComparator::Result result = func_comparator(func, reference_func);
+    ASSERT_TRUE(result.valid);
+}
+
+TEST(TransformationTests, SplitConvolutionWithFqVariant1TestSmallSize) {
```

Comment:
what is a difference between Variant1 and Variant2? Can these tests be renamed to contain some information regarding this difference?